store/bg/index.js

```diff
@@ -0,0 +1,298 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export const store = new Vuex.Store({
+  state: {
+    selectedMethod: '',
+    adding: [
+      {
+        name: 'splice',
+        shortDesc: 'елемент(и) в масив',
+        desc: 'Добавя и/или премахва елементи от масив.',
+        example: `arr.splice(2, 0, 'tacos');<br>
+        console.log(arr);`,
```

Comment:
Please remove console.log :)

Reply:
In this case console.log is not code is text to display
![captura de pantalla 2017-12-29 a las 17 34 03](https://user-images.githubusercontent.com/6000947/34441847-a350211a-ecbe-11e7-8084-50c0c7336391.png)

Reply:
For this to be complete it must also go even if it is the same in all languages.

Reply:
I understand the mechanism, Thank you @Gonzalo2310 .